front/lib/actions/mcp_internal_actions/servers/image_generation.ts

```diff
@@ -25,13 +33,15 @@ const createServer = (auth: Authenticator): McpServer => {
 
   server.tool(
     "generate_image",
-    "Generate an image in PNG format from a text prompt",
+    "Generate images from text descriptions. The more detailed and specific your prompt is, the" +
```

Comment:
Looks great. maybe let's use singular ("generate an image" vs images) given we don't expose the ability to generate more than 1 ?